api/src/test/java/org/pmiops/workbench/db/dao/DataSetServiceTest.java

```diff
@@ -272,4 +293,62 @@ public void testCapitalizeFirstCharacterOnly_emoji() {
     assertThat((DataSetServiceImpl.capitalizeFirstCharacterOnly("マリオに感謝しますが、私たちの王女は別の城にいます")))
         .isEqualTo("マリオに感謝しますが、私たちの王女は別の城にいます");
   }
+
+  @Test
+  public void testDomainToBigQueryConfig() {
+    mockLinkingTableQuery(ImmutableList.of("FROM `" + TEST_CDR_TABLE + ".person` person"));
+    final DataSetRequest dataSetRequest =
+        new DataSetRequest()
+            .conceptSetIds(Collections.emptyList())
+            .cohortIds(new ArrayList<>())
```

Comment:
Question: Why did you use new ArrayList here?

Reply:
It's just moved from `DataSetControllerTest`.

Reply:
Updated to use `Collections.emptyList()` everywhere in this test.